item_rater.py

```diff
@@ -60,12 +72,12 @@ def calculate_item_winrates(item_info, start_time, hero_name=None):
     return heroes
 
 
-def normalize_item_winrates_by_cost_and_hero_winrate(hero_table, item_info):
+def normalize_item_winrates_by_cost_and_hero_winrate(hero_table, item_info, min_num_games = 30):
```

Comment:
typically defaults in the function definition are provided without spaces, ie `foo(bar=4)`